cirq/testing/random_circuit.py

```diff
@@ -70,13 +75,15 @@ def random_circuit(qubits: Union[Sequence[ops.Qid], int],
         gate_domain = DEFAULT_GATE_DOMAIN
     if not gate_domain:
         raise ValueError('gate_domain must be non-empty')
-    max_arity = max(gate_domain.values())
 
     if isinstance(qubits, int):
         qubits = tuple(ops.NamedQubit(str(i)) for i in range(qubits))
     n_qubits = len(qubits)
     if n_qubits < 1:
         raise ValueError('At least one qubit must be specified.')
+    if n_qubits == 1:
```

Comment:
actually now that I think about this, we should probably just make it select the gates that are less than the number of qubits.

gate_domain = {key: val for key, val in gate_domains.itmes() if val <= n_qubits}